roles/openshift_sanitize_inventory/tasks/deprecations.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Check for usage of deprecated variables
+  debug: msg="{{ __deprecation_header }} {{ item }} is a deprecated variable and will be no longer be used in the next minor release. Please update your inventory accordingly."
+  when:
+  - lookup('env', item) | bool
+  with_items: "{{ __warn_deprecated_vars }}"
```

Comment:
Cool, could there be a pause after showing this (if any vars were flagged)? Otherwise it will fly past in the output and no one will see it...

Another thought: have we refactored out all the multiple includes of `openshift_sanitize_inventory`? Wouldn't want to pause multiple times. If it does still get included multiple times, we could just singleton-ize it to skip after the first time.